src/Core/GenericEqualityComparer.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
```

Comment:
- License header is missing
- Imports should go inside the namespace declaration below